enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -281,8 +282,21 @@ func (c *podmanCommandContainer) getPodmanRunArgs(workDir string) []string {
 	} else if c.options.User != "" && userRegex.MatchString(c.options.User) {
 		args = append(args, "--user="+c.options.User)
 	}
-	if strings.ToLower(c.options.Network) == "off" {
-		args = append(args, "--network=none")
+	networkMode := c.options.DefaultNetworkMode
+	networkProp := strings.ToLower(c.options.Network)
```

Comment:
can you switch directly on strings.ToLower(c.options.Network)?

Reply:
Done